### DISCUSSION TITLE
Looking for Collaborators / Help / This is an urgent

### DISCUSSION THREAD

**Discussion by mlmbusiness83** [community/community#133141] [Programming Help] [answered]
### Body

I have a AI ecosystem i built name Apex-Echo-AI. I integrated into a Multi-Vendor MarketPlace. I will give away a free copy of Apex-Echo-AI for your help and collaboration with testing and help launching to the hosting provider. If you would like to know more, please reach out to me. This is an urgent matter.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment, marked as answer]
Hi @mlmbusiness83, thanks for sharing with the community!

Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
